rd-net/RdFramework/Impl/RdMap.cs

```diff
@@ -192,7 +192,7 @@ public override void OnWireReceived(UnsafeReader stream)
               
               ReceiveTrace?.Log($"{this} :: {kind} :: key = {key.PrintToString()}" +
                                  (msgVersioned ? " :: version = " + version : "") +
-                                 $" :: value = {value.PrintToString()}"
+                                 $" :: value = {value?.PrintToString()}"
```

Comment:
`PrintToString` is supposed to handle `null` better (by printing `"<null>"`), so could we leave this as is?

Reply:
Nice catch, fixed